tests/Doctrine/Tests/ODM/PHPCR/Functional/Tools/Helper/TranslationConverterTest.php

```diff
@@ -451,35 +454,32 @@ public function testTranslateStrict()
         $this->assertFalse($this->session->hasPendingChanges());
     }
 
-    /**
-     * @expectedException \Doctrine\ODM\PHPCR\Exception\InvalidArgumentException
-     * @expectedExceptionMessage To untranslate a document, you need to specify the previous translation strategy
-     */
     public function testUntranslateMissingPrevious()
     {
-        $class = 'Doctrine\Tests\Models\Blog\Comment';
+        $class = \Doctrine\Tests\Models\Blog\Comment::class;
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('To untranslate a document, you need to specify the previous translation strategy');
         $this->converter->convert($class, array('en'));
     }
 
-    /**
-     * @expectedException \Doctrine\ODM\PHPCR\Exception\InvalidArgumentException
-     * @expectedExceptionMessage need to specify the fields that where previously translated
-     */
     public function testUntranslateMissingFields()
     {
-        $class = 'Doctrine\Tests\Models\Blog\Comment';
+        $class = \Doctrine\Tests\Models\Blog\Comment::class;
```

Comment:
`use` would be nice.

Reply:
there are two different comment classes used in this test :-/ i now made an alias for this one.